src/snowflake/cli/api/output/types.py

```diff
@@ -79,6 +85,25 @@ def _prepare_payload(self, cursor):
         return None
 
 
+class AggregatedResult(CollectionResult):
+    """
+    Prints results from many cursors as single result. Cursors must have the same number and order of columns.
+    """
```

Comment:
What will happen if this is not true? Should we print a warning / fail in runtime?

Reply:
It is even more complicated, so I decided to use MultipleResult.